ui/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "vscode-azureextensionui",
     "author": "Microsoft Corporation",
-    "version": "0.38.3",
+    "version": "0.38.4",
```

Comment:
We consider updating the VS Code types to be a "breaking change" so we'd need to bump this to "0.39.0". Also, is it worth requiring 1.52 for all our extensions just for "MarkdownString"? I lean towards no since 1.52 is so recent, but 🤷‍♂️ Even if we say "no", maybe you can work some magic to get `tooltip` to work as a MarkdownString (if the user has 1.52+) without "officially" updating the VS Code types?

Reply:
That makes sense. Since all this library is doing is copying from the `AzExtTreeItem` into `TreeItem`, it's probably possible to do this without changing the VSCode dependency. We do plan to use Markdown though: https://github.com/microsoft/vscode-docker/issues/2592

I'll look into whether or not it's possible to do this with magic instead of a dependency update. :smile:

Reply:
Seems like it worked fine so I rolled that stuff back out.